Day6_excercise.rb

```diff
@@ -0,0 +1,77 @@
+#1.store names of students
+names = ["Jessie", "Wendy", "Kelly"]
+#2.store students' names and their ages
+students = {Jessie: 29, Wendy: 22, Kelly: 33}
```

Comment:
Do you need to add quotes to students' names?

Reply:
Yes

Reply:
Okay, please fix it then.